graph/pull.go

```diff
@@ -25,6 +30,7 @@ type Puller interface {
 	Pull(tag string) (fallback bool, err error)
 }
 
+// NewPuller returns a new implementation conforming to Puller interface.
```

Comment:
don't think "implementation" is the right word here, maybe "instance" or "object"?